src/docs/stan-reference/examples.tex

```diff
@@ -7145,87 +7193,325 @@ \section{Fitting a Gaussian Process}\label{fit-gp.section}
 block because it involves unknown parameters and thus can't simply be
 precomputed as transformed data.  The rest of the model consists of
 the priors for the hyperparameters and the multivariate
-normal likelihood, only now the value \code{y} is known and the
+Cholesky-parameterized normal likelihood, only now the value \code{y} is known and the
 covariance matrix \code{Sigma} is an unknown dependent on the
-hyperparameters.  Note that a prior is put on the scale term
-\code{inv\_rho\_sq}, then \code{rho\_sq} is defined as
-\code{inv(inv\_rho\_sq)}, which is just a more efficient way of
-expressing \code{1~/~rho\_sq}.  An alternative would be to just divide
-by \code{inv\_rho\_sq}.
+hyperparameters, allowing us to learn the hyperparameters. 
+
+We have used the Cholesky parameterized \distro{MultiNormal} rather than the
+standard \distro{MultiNormal} because it allows us to the
+\code{cholesky\_decompose} function which has been optimized for both small and
+large matrices. When working with small matrices the differences in
+computational speed between the two approaches will not be noticeable, but for
+larger matrices ($N \gtrsim 100$) the Cholesky decomposition version will be faster.
```

Comment:
For a single draw from multi_normal, is this true? Isn't multi_normal just doing a Cholesky internally to do the solve?